ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/recovery/AtomicActionRecoveryModule.java

```diff
@@ -214,6 +216,7 @@ private void processTransactionsStatus()
        // to recover anything but if this module is still configured it would 
        // get an NPE
         if (_transactionUidVector != null) {
+
```

Comment:
I don't know if this formatting was intentional or not to add? I think it could be reverted if you don't object?

Reply:
Yep, I didn't see it. Thanks